src/launcher.ts

```diff
@@ -189,27 +189,9 @@ export default class VSCodeServiceLauncher extends ChromedriverServiceLauncher {
      * @returns "insiders" if `desiredReleaseChannel` is set to this otherwise a concrete version
      */
     private async _setupChromedriver (desiredReleaseChannel: string) {
-        let version = await this._fetchVSCodeVersion(desiredReleaseChannel)
+        const version = await this._fetchVSCodeVersion(desiredReleaseChannel)
 
         try {
-            /**
-             * if testing insiders fetch from latest release rather than from main
-             * branch given that there could be a situation where the VSCode team
-             * updates Electron but haven't released a stable VSCode version yet causing
-             * the service to download a wrong Chromedriver version
```

Comment:
I think this will be very infrequent as: 
A: The VS Code team does not upgrade Electron often. 
B: Insiders is released at least daily from `main`.

Reply:
Running a quick `vs code release notes "Chromium"` search gives hits against

1. June 22
2. March 22
3. September 21
4. July 21
5. April 21

Reply:
From the API:

| Version | Driver |
| -- | -- | 
| 1.69.0 | 100.0.4896.160 |
| 1.68.0 |98.0.4758.141
| 1.67.0 |98.0.4758.141 |
| 1.66.0 |98.0.4758.109 |
| 1.65.0 |91.0.4472.164 |
| 1.64.0 |91.0.4472.164 |
| 1.63.0 | 91.0.4472.164 |
| 1.62.0 | 91.0.4472.164 |
| 1.61.0 | 91.0.4472.164 |
| 1.60.0 | 91.0.4472.164 |
| 1.59.0 | 91.0.4472.124 |
| 1.58.0 | 89.0.4389.128 |
| 1.57.0 | 89.0.4389.128 |
| 1.56.0 | 89.0.4389.114 |